src/Marten/Schema/DuplicatedField.cs

```diff
@@ -14,18 +14,20 @@ namespace Marten.Schema
     public class DuplicatedField: Field, IField
     {
         private readonly Func<Expression, object> _parseObject = expression => expression.Value();
+        private readonly StoreOptions _storeOptions;
         private readonly bool useTimestampWithoutTimeZoneForDateTime;
         private string _columnName;
 
-        public DuplicatedField(EnumStorage enumStorage, MemberInfo[] memberPath, bool useTimestampWithoutTimeZoneForDateTime = true, bool notNull = false)
-            : base(enumStorage, memberPath, notNull)
+        public DuplicatedField(StoreOptions storeOptions, MemberInfo[] memberPath, bool useTimestampWithoutTimeZoneForDateTime = true, bool notNull = false)
```

Comment:
This is breaking change. `DuplicatedField` in theory should be internal detail, but as this is a popular feature then IMHO it might break our users' code.
I'd suggest adding a new constructor with store options parameter and keep this one calling newly added with the current usage.

Reply:
I thought about this at the time, but figured I'd just push the code and see what you thought; I'll go with my gut next time ;)

Reply:
Normally it would be a good idea to not have redundant code, but we need to be careful for the breaking changes. In v4 we should definitely make internal as much as we can, then such refactoring would be much easier.